.gitignore

```diff
@@ -123,6 +123,7 @@ dist
 .env.development.local
 .env.test.local
 .env.production.local
+/.vscode
```

Comment:
Why do we need symbol "/" here?

Reply:
Because .vscode is a folder, not a file or it's unnecessary even if .vscode is a folder?

Reply:
Change /.vscode to .vscode